runelite-client/src/main/java/net/runelite/client/plugins/tearsofguthix/TearsOfGuthixPlugin.java

```diff
@@ -112,4 +122,68 @@ public void onDecorativeObjectDespawned(DecorativeObjectDespawned event)
 		DecorativeObject object = event.getDecorativeObject();
 		streams.remove(object);
 	}
+
+	//Shows the player's lowest experience skill
+	@Subscribe
+	public void onGameTick(GameTick gameTick)

```

Comment:
You can probably just do this in GameStateChanged event

Reply:
When the player logs in already at tears of guthix the onGameStateChanged doesnt seem to be triggered. Works fine when teleporting to and from the zone, but not when logging in at it. GameTick works for this though but perhaps there are other better solutions?